<launch>
    <node
        pkg="road_layout_estimation"
        type="visual_odometry_fake"
        name="visual_odometry_fake"
        respawn="true"
        output="screen"
    />
    <node
        pkg="road_layout_estimation"
        type="road_layout_estimation"
        name="road_layout_estimation"
        required="true"
        output="screen"
        args="/visual_odometry/odometry_no_error"
    />
    <node
        pkg="rviz"
        type="rviz"
        name="rviz"
        respawn="true"
        args="-d /home/limongi/workspace_ros/src/road_layout_estimation/rviz/road_layout_manager.rviz"
        output="screen"
    />

    <node
        pkg="rqt_reconfigure"
        type="rqt_reconfigure"
        name="rqt_reconfigure"
    />
</launch>
